apps/web/src/services/iam/components/UserManagementAddModal.vue

```diff
@@ -34,18 +30,27 @@ import UserManagementAddPassword from '@/services/iam/components/UserManagementA
 import UserManagementAddRole from '@/services/iam/components/UserManagementAddRole.vue';
 import UserManagementAddTag from '@/services/iam/components/UserManagementAddTag.vue';
 import UserManagementAddUser from '@/services/iam/components/UserManagementAddUser.vue';
+import { useRoleBindingCreateMutation } from '@/services/iam/composables/use-role-binding-create-mutation';
+import { useUserCreateMutation } from '@/services/iam/composables/use-user-create-mutation';
+import { useWorkspaceGroupAddUsersMutation } from '@/services/iam/composables/use-workspace-group-add-users-mutation';
+import { useWorkspaceUserCreateMutation } from '@/services/iam/composables/use-workspace-user-create-mutation';
 import { USER_MODAL_TYPE } from '@/services/iam/constants/user-constant';
 import { checkEmailFormat } from '@/services/iam/helpers/user-management-form-validations';
 import { useUserPageStore } from '@/services/iam/store/user-page-store';
 import type { AddModalMenuItem, AddAdminRoleFormState } from '@/services/iam/types/user-type';
 
 
+
 const userPageStore = useUserPageStore();
 const userPageState = userPageStore.state;
 const domainStore = useDomainStore();
 
 const route = useRoute();
 
+const queryClient = useQueryClient();
+const { key: userListQueryKey } = useServiceQueryKey('identity', userPageState.isAdminMode ? 'user' : 'workspace-user', 'list');
```

Comment:
Nitpick: this is not going to work as expected! reactivity assessment of variables should occur within reactivity context.
```suggestion
const { key: userListQueryKey } = useServiceQueryKey('identity', 'user', 'list');

const { key: userListQueryKey } = useServiceQueryKey('identity', 'workspace-user', 'list');
```